content/ux-guide/research/accessibility.md

```diff
@@ -0,0 +1,120 @@
+permalink: /ux-guide/research/accessibility/
+layout: layouts/page
+tags: ux-guide
+eleventyNavigation: 
+  parent: ux-guide-research
+  key: ux-guide-accessibility
+  order: 7
+  title: Accessibility
+sidenav: true
+sticky_sidenav: true
+subnav:
+- text: Conducting research with people with disabilities
+  href: '#conducting-research-with-people-with-disabilities'
+- text: Supporting neurodiversity in research
+  href: '#supporting-neurodiversity-in-research'
+- text: Further reading
+  href: '#further-reading'
+---
+
+Our primary goal as designers is to ensure our products work well for everyone. We have a [legal]({{ '../../research/legal/' | url }}) and [ethics]({{ '../../research/ethics/' | url }}) responsibility to include people with disabilities in our research, and to ensure their needs are supported during the process.
+
+**At least one in four Americans live with a disability.** Making our research practices accessible means being inclusive for people who:
+
+- Use assistive technology (such as screen readers, keyboard-only navigation, - speech recognition software)
+- Use mobility devices (wheelchairs, canes, or walkers, for example)
+- Have difficulty living independently or managing daily tasks
+- Are [neurodivergent or have a cognitive disability](#supporting-neurodiversity-in-research)
+- Are deaf or have hearing loss
+- Are blind or have low vision
+
+While these conditions are protected under the Americans with Disabilities Act (ADA), they may not be seen as impairments by people within those communities. [Be respectful and curious](https://www.nih.gov/about-nih/what-we-do/science-health-public-trust/perspectives/writing-respectfully-person-first-identity-first-language) when writing or asking about people’s unique needs.
+
+[Accessibility](https://guides.18f.gov/accessibility/) benefits everyone, including people with slow internet connections, changing abilities, caregiving responsibilities, and temporary limitations from illness or injury. Centering accessibility helps us achieve our broader [mission](https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services) of delivering inclusive digital services to the public.
+
+## Conducting research with people with disabilities
+
+Here are some ways to support people with disabilities during the research process:
+
+- **Offer accommodations throughout the process.** We have suggested language in the [screener questions](https://airtable.com/appkBrEBVTMd9M5VC/pagxzSXmHxmvl7u8O?SlYml=allRecords&JVZOB=recyNUVODFwrEr7EZ) and [consent form](https://guides.18f.gov/ux-guide/research/do/#getting-informed-consent) templates. Check in with participants at the beginning of the session to see if they need anything or if there’s anything you should know before you get started. This gives them a chance to talk about what’s going on for them and add anything they’ve been thinking about.
```

Comment:
@nicoleslaw One of the changes in discussion for the ux guide process is to not including links to internal resources. Links in the UX guide should link to publicly viewable resources. 




Please remove "We have suggested language in the [screener questions](https://airtable.com/appkBrEBVTMd9M5VC/pagxzSXmHxmvl7u8O?SlYml=allRecords&JVZOB=recyNUVODFwrEr7EZ)"

Reply:
Also, if this section is about offering accommodations, can you help me understand why the consent form should be referenced? Is it because we asking for permission? I'm not sure its offering specific accommodations.